src/main/java/domain/GenerateMaze.java

```diff
@@ -0,0 +1,134 @@
+package domain;
+
+import bootstrap.Driver;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class GenerateMaze {
+    private int id=0;
+    private String[] configValues;
+    private int[][] maze;
+    private String difficulty;
+    private int size;
+    private List<Integer> walls = new ArrayList<>();
+
+    public GenerateMaze(String[] args){
+        this.configValues = args;
```

Comment:
We are passing all arguments to this function and then using Driver.getArgs to get the arguments. If thats the intention, then you do not need to pass in args.